src/main/java/net/minecraftforge/forgespi/locating/IModFile.java

```diff
@@ -34,6 +34,6 @@
     IModFileInfo getModFileInfo();
 
     enum Type {
-        MOD, LIBRARY, LANGPROVIDER
+        MOD, LIBRARY, LANGPROVIDER, GAMELIBRARY
```

Comment:
Can we please add javadocs to explain what these are, and why library/game library are different.